### PR TITLE
Submit current step form when clicking on a next step title

### PR DESCRIPTION
**Pull request by maximebiloe**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Submit the current step form when clicking on a next step title. Also, when you change your delivery address after having selected your carrier, it has to unvalidate the carrier step. |
| Type? | bug fix |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-929 |
| How to test? | Look at the forge ticket 😉 |

### CODE REVIEW

File: themes/classic/assets/js/theme.js [hunk 1]
```diff
@@ -150,7 +150,7 @@
 /***/ function(module, exports, __webpack_require__) {
 
 	var __WEBPACK_AMD_DEFINE_ARRAY__, __WEBPACK_AMD_DEFINE_RESULT__;/*!
-	 * jQuery JavaScript Library v2.2.4
+	 * jQuery JavaScript Library v2.2.3
```
Comment (Quetzacoalt91) [inline themes/classic/assets/js/theme.js]: Is the rollback of jQuery expected ?

Comment (maximebiloe) [inline themes/classic/assets/js/theme.js]: 😢 nope

Comment (maximebiloe) [inline themes/classic/assets/js/theme.js]: Updated 😄

**Comment by vincentbz** [comment]
tested and ok, thanks !
shipping price and methods are updated, the good address is taken when order completed.

**Comment by mickaelandrieu** [comment]
Thank you @maximebiloe !

**Pull request merged**